src/Foundatio.TestHarness/Caching/CacheClientTestsBase.cs

```diff
@@ -1036,7 +1094,7 @@ public virtual async Task MeasureThroughputAsync()
                 await cache.SetAsync("test", 13422);
                 await cache.SetAsync("flag", true);
                 Assert.Equal(13422, (await cache.GetAsync<int>("test")).Value);
-                Assert.Null(await cache.GetAsync<int>("test2"));
+                Assert.False((await cache.GetAsync<int>("test2")).HasValue);
```

Comment:
These tests were skipped so I think this is just something that was broken as GetAsync always returns a cache value and never null